product-entry.js

```diff
@@ -0,0 +1,40 @@
+import pokemonArray from './pokemonArray.js';
+const myForm = document.getElementById('my-form');
+
+myForm.addEventListener('submit', (event) => {
```

Comment:
The spacing in the EventListener makes it a bit difficult for me to read. I encourage you to reduce your extra returns between lines in if statements and const declarations. The logic looks good tho!